ha/const.py

```diff
@@ -13,9 +13,14 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-HA_INIT_DIR="/var/cortx/ha/"
 RA_LOG_DIR="/var/log/seagate/cortx/ha"
+PCSD_LOG="/var/log/pcsd/pcsd.log"
+PACEMAKER_LOG="/var/log/pacemaker.log"
+EXTRA_CMD_LOG="/var/log/seagate/cortx/ha/extra_cmd_logs"
```

Comment:
Name it ha_cmds_output.

Reply:
@ajay-paratmandali Did we find out and inform the component which add the pacemaker and corosync logs to support bundle currently?

Reply:
@indrajitzagade no i will check with hare team today

Reply:
ok

Reply:
Done